js/views/inbox_view.js

```diff
@@ -84,6 +84,14 @@
                     .removeClass('android-dark')
                     .removeClass('android')
                     .addClass(theme);
+
+            // Need to add this to <body> as well, so debug log dialog is themed.
+            //   (Can't use only this location for the theme class, because the 'index'
+            //   class on our <div> overrides it).
+            $(document.body).removeClass('ios')
```

Comment:
I find it a bit surprising for a view to operate outside its own element. Perhaps theming should be moved to AppView instead, which binds to the body element. If the index class is interfering it sounds like a slight style refactor is needed.